dbt-tests-adapter/pyproject.toml

```diff
@@ -31,14 +31,6 @@ dependencies = [
     "dbt-adapters",
     "pyyaml",
 ]
-
-[project.optional-dependencies]
```

Comment:
This was moved down into the `hatch` environment dependencies elsewhere. This aligns with that decision.